buildpack.md

```diff
@@ -967,6 +969,8 @@ The buildpack ID:
 
 If an `order` is specified, then `stacks` MUST NOT be specified.
 
+Buildpack API MUST be in form `<major>.<minor>` and describe the implemented buildpack API. When `<major>` is `0`, this buildpack is only compatible with lifecycles implementing that exact buildpack API. When `<major>` is greater than `0`, this buildpack is compatible with lifecycles implementing buildpack API `<major>.<minor>`, where `<major>` equals the `<major>` of the buildpack and `<minor>` is greater than or equal to that of the buildpack.
```

Comment:
My understanding is that we also intend to support `api = "<major>"` (where `<major>` > 0), which is equivalent to `api = "<major>.0"`.